pipelines/internal/commands/run/run.go

```diff
@@ -757,3 +762,13 @@ func sshDebug(project string) *genomics.Action {
 		Flags:        []string{"RUN_IN_BACKGROUND"},
 	}
 }
+
+func periodicLogs(arguments ...string) *genomics.Action {
```

Comment:
This is just bash() with the background flag set I think

Reply:
The bash command has a && separator which doesn't work for this command. I could change the bash command to receive a separator instead.

Reply:
Used bash with constructed string